CHANGELOG.md

```diff
@@ -4,6 +4,17 @@
 
 ### Features / Improvements 🚀
 
+- N/A
```

Comment:
Hi @thaddmt. Thanks for finishing the release with the nanoid fix. I think the version in the changelog should be 1.3.1.

Reply:
Ah shoot, will fix that thanks for the catch :)